ansible/roles/elasticsearch/tasks/main.yml

```diff
@@ -82,6 +82,10 @@
     enabled: yes
   become: true
 
+- name: remove jndi plugin from log4j jar to mitigate log4shell
+  shell: zip -q -d /usr/share/elasticsearch/lib/log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
```

Comment:
Wondering if instead of modifying the jar that is there we could insead delete the log4j jar and copy in a new version.  Not sure if this would work.

Reply:
Thanks @keith-turner - this is actually a [published](https://logging.apache.org/log4j/2.x/security.html) and accepted mitigation per the log4j team themselves. I don't think it is worthwhile trying anything else out given the ELK stack is completely unrelated to our primary focus here in Accumulo.

![image](https://user-images.githubusercontent.com/16342666/147983787-98b43e41-0e90-46c8-aabf-18053cf9df59.png)